src/components/Card/Card.spec.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Card from './Card';
+
+describe('Card', () => {
+	it('renders category,name and address', () => {
+		render(
+			<Card
+				entries={{
+					id: '1234',
+					category: 'Flohmarkt',
+					name: 'Faust',
+					address: 'Zur Bettfedernfabrik 3, 30451 Hannover',
+				}}
+			/>
+		);
+
+		const category = screen.getByText(/Flohmarkt/i);
+		const name = screen.getByText(/Faust/i);
```

Comment:
Since you use a `h2` element for this, you could try `getByRole('heading', {name: /Faust/i})`; this query is the number 1 priority [have a look here](https://testing-library.com/docs/queries/about#priority).